### ISSUE TITLE
Chapter 10 Error ( Accuracy is always 18.18% i.e constant)

### ISSUE THREAD

**Issue by Avi-141** [PacktPublishing/PyTorch-Computer-Vision-Cookbook#7]
I have trained your model on the hmdb51 dataset using a lesser number of classes but the accuracy always remains contant even for different learning rates or a different lr_scheduler. 

Please guide me further.

**Comment by mave5** [comment]
can you provide more details? how does your training loss look like?
